packages/beacon-node/src/util/fullOrBlindedBlock.ts

```diff
@@ -0,0 +1,221 @@
+import {ChainForkConfig} from "@lodestar/config";
```

Comment:
it's important to have unit tests for both deneb + electra especially when this [consensus PR](https://github.com/ethereum/consensus-specs/pull/3875/files) is merged

recommend to wait for the electra PR to be merged to `unstable` first https://github.com/ChainSafe/lodestar/pull/6986